### ISSUE TITLE
Taking Escape Shuttle as a Head during Revround Shows "fleeing the station" 

### ISSUE THREAD

**Issue by SinfulBliss** [tgstation/tgstation#58924]
[image]https://user-images.githubusercontent.com/16496315/117375839-82a2ce80-ae95-11eb-9432-4ab1ae89a8a4.png[/image]
if you win as a head during rev round, and take escape shuttle, it says you "fled the station"

**Comment by ATH1909** [comment]
Did you leave the station's z-level in any way before taking the escape shuttle that round? IIRC, leaving the station's z-level as a head of staff will permanently mark you as having been exiled, even if you later return to the station.

**Comment by Serxule** [comment]
yes because you left the station without every rev being dead or deconverted, this is intended

**Comment by interestingusernam3** [comment]
It's correct tho.

**Comment by legoscape** [comment]
Shouldn't this be a bug though? 

Revolutionaries are a hostile environment factor that must be cleared before the shuttle leaves.

**Comment by interestingusernam3** [comment]
Succesfully reproduced in R175086.

https://tgstation13.org/parsed-logs/terry/data/logs/2021/12/19/round-175086/round_end_data.html
